nifi-ignite-extensions/nifi-ignite-services/src/main/java/pl/touk/nifi/ignite/typing/IgniteTyping.java

```diff
@@ -0,0 +1,36 @@
+package pl.touk.nifi.ignite.typing;
+
+import org.apache.nifi.serialization.record.DataType;
+import org.apache.nifi.serialization.record.RecordField;
+import org.apache.nifi.serialization.record.RecordFieldType;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class IgniteTyping {
+
+    private static final Map<String , DataType> IGNITE_TO_NIFI = new HashMap<String , DataType>() {{
+        put("byte", RecordFieldType.BYTE.getDataType());
+        put("short", RecordFieldType.SHORT.getDataType());
+        put("int", RecordFieldType.INT.getDataType());
+        put("long", RecordFieldType.LONG.getDataType());
+        put("boolean", RecordFieldType.BOOLEAN.getDataType());
+        put("float", RecordFieldType.FLOAT.getDataType());
+        put("double", RecordFieldType.DOUBLE.getDataType());
+        put("decimal", RecordFieldType.DECIMAL.getDataType());
+        put("String", RecordFieldType.STRING.getDataType());
+        put("Date", RecordFieldType.DATE.getDataType());
+        put("Timestamp", RecordFieldType.TIMESTAMP.getDataType());
+        put("Time", RecordFieldType.TIME.getDataType());
+    }};
+
+    public static final String SUPPORTED_IGNITE_TYPE_NAMES = String.join(",", IGNITE_TO_NIFI.keySet());
+
+    public static RecordField get(String binaryFieldTypeName) throws IllegalArgumentException {
```

Comment:
is it called anywhere?

Reply:
currently no.
it can be in fact moved to nifi-services,

in future if we'll build ignite lookup based on java client it might be useful .. but yes it's yagni

Reply:
ah, it shouldn't be here, in this PR,
to be removed.
thanks!

Reply:
35bf388a88e05ef1254e607a0bd589239149dc5d